libraries/AP_Motors/AP_Motors_Class.cpp

```diff
@@ -79,26 +79,32 @@ void AP_Motors::set_radio_passthrough(float roll_input, float pitch_input, float
 }
 
 /*
-  write to an output channel
+ * write to an output channel
+ * pwm is in usec with range [1000,2000] usec
```

Comment:
I'm not sure if that would be a useful description. I was just pointing out that 1000-2000 range isn't correct and so that specific part shouldn't be there.